tusclient/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.2'
+__version__ = '1.0.3'
```

Comment:
Because we are dropping support for older versions of python, this arguable should be a major (2.0.0) or minor (1.1.0) change rather than than a patch (1.0.x) change.